RELEASE.md

```diff
@@ -86,7 +86,7 @@ Check the [dependencies GitHub label](https://github.com/prometheus/prometheus/l
 This bot currently does not manage `+incompatible` and `v0.0.0` in the version specifier for Go modules.
 
 Note that after a dependency update, you should look out for any weirdness that
-might have happened. Such weirdnesses include but are not limited to: flaky
+might have happened. Such weirdness include but are not limited to: flaky
```

Comment:
Should we say includes then?

Reply:
Yes, either "weirdnesses include but are" or "weirdness includes but is". But TBH the existing wording appears fine to me. It's a bit playful, but I don't see anything wrong with that. If you want a more compact and technical wording, I would probably do the following:

"Note that after a dependency update, you should look out for any weirdness that might have happened, including but not limited to flaky tests, differences in resource usage, and panics."